x-pack/plugins/apm/common/service_nodes.ts

```diff
@@ -5,4 +5,12 @@
  * 2.0.
  */
 
+import { UNIDENTIFIED_SERVICE_NODES_LABEL } from './i18n';
+
 export const SERVICE_NODE_NAME_MISSING = '_service_node_name_missing_';
```

Comment:
Shouldn't `UNIDENTIFIED_SERVICE_NODES_LABEL` and `SERVICE_NODE_NAME_MISSING` be in the same file?

I think it makes most sense to move `UNIDENTIFIED_SERVICE_NODES_LABEL` to this file.